src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const DISCOUNT_20$ = 20;
```

Comment:
Variable names should not contain symbols, like '$'. And try to void numbers in names. Rename to `SHORT_TERM_DISCOUNT` maybe. Or maybe you have a better option